### DISCUSSION TITLE
How to Restore Excluded Album Art to Album List

### DISCUSSION THREAD

**Discussion by saread3** [rern/rAudio#1802] [General]
I accidentally excluded some album art from the album listing and for the life of me I can't figure out how to get it back. The art shows up in other lists like Latest or Album Artist.  I can't seem to get it back into the main Album list.

**Comment by rern** [comment]
- Remove the albums from `/srv/http/data/mpd/albumignore`
- Update Library

**Reply by saread3** [reply]
Thx.
